app/views/search/_whats_cwic_modal.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="modal fade" id="whats-cwic-modal" tabindex="-1" role="dialog" aria-labelledby="whats-cwic-modal-title">
```

Comment:
I'm going to move this into the renderer. The purpose of renderer plugins is to hold these sort of one-off things that would otherwise bloat the page.